packages/@react-spectrum/picker/stories/Picker.stories.tsx

```diff
@@ -63,6 +63,19 @@ storiesOf('Picker', module)
       </Picker>
     )
   )
+  .add(
+    'can be deselected',
+    () => {
+      const [selected, setSelected] = React.useState<React.Key>();
+      return (
+        <Picker label="Test" selectedKey={selected} onSelectionChange={val => setSelected(val === selected ? null : val)}>
```

Comment:
I realize this looks different than the rest of the stories, but couldn't find a better way to do this. If you have suggestions let me know. Thanks.